sidebars.js

```diff
@@ -24,13 +24,13 @@ module.exports = {
     },
     {
       type: 'category',
-      label: 'Technical guide',
+      label: 'app.opensauced.pizza',
```

Comment:
just the label is fine, the items don't have to actually be moved around :D

Reply:
@0-vortex Sorry I didn't get you. You mean, should I change label to `app.opensauced.pizza` and keep folder name as `technical` as it is, right?

Reply:
yes, at the moment the folder is being added 3 times to the PR